### PR TITLE
libxdg-basedir 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by dirkvdb**
Formula for the libxdg-basedir library which is an implementation of the XDG Base Directory specifications.

### CODE REVIEW

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by mikemcquaid** [comment]
Please fix the `brew audit --strict libxdg-basedir` warnings: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26856/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_libxdg_basedir___strict/

**Comment by dirkvdb** [comment]
Issues should be fixed now

File: Library/Formula/libxdg-basedir.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  url "http://github.com/devnev/libxdg-basedir/archive/libxdg-basedir-1.2.0.tar.gz"
+  sha256 "1c2b0032a539033313b5be2e48ddd0ae94c84faf21d93956d53562eef4614868"
+  desc "An implementation of the XDG Base Directory specifications"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh", "--disable-dependency-tracking",
```
Comment (DomT4) [inline Library/Formula/libxdg-basedir.rb]: Is this always the case? There seems to be an autogen and a configure file in the upstream repo. What's stopping us using the configure script for stable builds?

Comment (dirkvdb) [inline Library/Formula/libxdg-basedir.rb]: There is a configure.ac file in the repo but no configure file. So it is needed to run the autogen to generate the configure file.

File: Library/Formula/libxdg-basedir.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class LibxdgBasedir < Formula
+  homepage "https://github.com/devnev/libxdg-basedir"
+  url "http://github.com/devnev/libxdg-basedir/archive/libxdg-basedir-1.2.0.tar.gz"
+  sha256 "1c2b0032a539033313b5be2e48ddd0ae94c84faf21d93956d53562eef4614868"
+  desc "An implementation of the XDG Base Directory specifications"
```
Comment (DomT4) [inline Library/Formula/libxdg-basedir.rb]: Can you move this above the homepage? Thanks!

File: Library/Formula/libxdg-basedir.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/libxdg-basedir.rb]: :+1:. It'd be nice of upstream to release tarballs with `configure` scripts pre-generated so we can drop the autotool deps, but I don't have an issue with this setup.

Comment (dirkvdb) [inline Library/Formula/libxdg-basedir.rb]: True

File: Library/Formula/libxdg-basedir.rb (hunk 2 above)
Comment (dirkvdb) [inline Library/Formula/libxdg-basedir.rb]: Done

**Comment by bfontaine** [comment]
Can someone from @Homebrew/owners give their :+1: / :-1: here since it’s a new formula?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**